### PR TITLE
Fix/2305 stackable setting missing blocks

### LINKED ISSUE #2305: Stackable settings issue - V3 button blocks are missing in Settings > Enable & Disable Blocks and Getting Started tab
**Describe the bug**
In Enable & Disable Blocks settings and Getting Started tab, these blocks are missing:
- Essential Blocks:
	- Button
	- Icon Button
- Special Blocks:
	- Social Buttons
	- Load More
	- Pagination

**To Reproduce**
1. Go to Settings > Stackable
2. In Stackable settings, go to Enable & Disable Blocks
3. Check to see if button blocks listed from the description above is missing
4. Go to Getting Started tab
5. Check to see if button blocks listed from the description above is missing

**Expected behavior**
Button blocks should be included in Enable & Disable Blocks settings and Getting Started tab

**Screenshots**
**Enable & Disable Blocks settings**
image

**Getting Started tab**
image

### PR DESCRIPTION
**Pull request by vmc08**
Fixes #2305

### CODE REVIEW

**Review by bfintal — CHANGES_REQUESTED** [review]
We'll need to remove "Button Group" from the getting started and activate/deactivate list because there's really no way to add one by itself without adding a button or an icon button.

**Comment by andeng1106** [comment]
**Issue:**
- When disabling Button and Icon Button block, it still shows in block inserter and when using the /

https://user-images.githubusercontent.com/103395655/194788945-4ff9c1d8-27b6-41d1-aa9c-c132077374a4.mp4

**Question:**
- Load More and Pagination button seems to be still missing. Is it expected not to be included?

@vmc08 @bfintal

**Comment by andeng1106** [comment]
@vmc08 Tried the new build, the same issue still happens on my end.

**Comment by bfintal** [comment]
> * Load More and Pagination button seems to be still missing. Is it expected not to be included?

@andeng1106 Let's do this in a separate issue instead

File: src/welcome/admin.js [hunk 1]
```diff
@@ -27,44 +28,13 @@ import {
...
-			}
-		} )
-	} )
-	Object.keys( blocks ).forEach( type => {
-		blocks[ type ] = sortBy( blocks[ type ], 'name' )
-	} )
-	return blocks
-}
+const FREE_BLOCKS = importBlocks( require.context( '../block', true, /block\.json$/ ) )
+export const getAllBlocks = () => applyFilters( 'stackable.settings.merge-blocks', FREE_BLOCKS )
```
Comment (bfintal) [inline src/welcome/admin.js:36]:
```suggestion
export const getAllBlocks = () => applyFilters( 'stackable.settings.blocks', FREE_BLOCKS )
```

**Pull request merged**